lisp/ess-mode.el

```diff
@@ -311,18 +311,10 @@ indentation style. See `ess-style-alist' for predefined styles."
   ;; completion
   (add-hook 'comint-dynamic-complete-functions 'ess-complete-filename nil 'local)
   (delq t comint-dynamic-complete-functions)
-  (set (make-local-variable 'comint-completion-addsuffix)
-       (cons "/" ""))
-  (add-hook 'hack-local-variables-hook #'ess--set-style-in-buffer nil t)
+  (setq-local comint-completion-addsuffix (cons "/" ""))
+  (add-hook 'hack-local-variables-hook (lambda() (ess-set-style ess-style t t)) nil t)
```

Comment:
Can we keep the named function? Named functions are much easier to debug in hooks than lambdas.

Reply:
I know but it seems like a waste for a single call.